bundled-app/README.md

```diff
@@ -0,0 +1,12 @@
+# Roadmap for Bundled Application
+
+A large majority of interactive notebook users use notebooks against a
+traditional SciPy stack. Sometimes installing this talk can be difficult.
+Although tools like `conda` make the process easier it can still be a
+hassle for some users. Our bundled kernel intiative aims to make this
+easier.
```

Comment:
The main drive here is to make the barrier to entry really low. Install and run one thing, show people the way to get more complex later on.